main.cpp

```diff
@@ -0,0 +1,33 @@
+// This is a program that calculates stock information
+// Created by NAOMI BROWN on 1/31/16.
+
+#include <iostream>
+#include <iomanip>
+
+// Constants
+const int SHARES_BOUGHT = 1000;
+const int SHARES_SOLD = 1000;
+const double COMMISSION_RATE = .02;
+const double PURCHASE_PRICE = 45.50;
+const double SALE_PRICE = 56.90;
+
+// Variables
+double totalPurchasePrice = SHARES_BOUGHT*PURCHASE_PRICE;
+double totalBuyCommission = totalPurchasePrice * COMMISSION_RATE;
+double totalSalePrice = SHARES_SOLD*SALE_PRICE;
+double totalSaleCommission = totalSalePrice*COMMISSION_RATE;
+double totalProfit = (totalSalePrice-totalSaleCommission)-(totalPurchasePrice+totalBuyCommission);
```

Comment:
Since these variables are part of our program's scope we should not escalate them to the parent scope.

TL;DR;
I would bring them into main and keep them as close to their use as possible.

Consider this, since constants are immutable there is no real danger of them being accessible in the global scope aside from name collisions since we are not using our own namespace currently.

But the other variables live in a public space and would be accessible anywhere in the program and mutable. This could lead to them being changed in the future without us knowing leading to a bug here.

While this program is not likely to expand the most important job of a developer is to build things can handle change. All software changes all the time. So our job is to control unintended access to the memory defined in our logical operations.

Basically don't provide access to any variables to anything higher than it absolutely needs and if it needs to be higher make sure that access is __READ ONLY__